internal/c/start.go

```diff
@@ -41,6 +42,42 @@ func waitTimeout(
 	}
 }
 
+func sendChildNotification(
```

Comment:
The name threw me off. It is a little ambiguous. Would you consider `sendNotificationToSupervisor()`?

Reply:
Good idea!